docs/_includes/arguments/timeline.md

```diff
@@ -1,10 +1,10 @@
 ### Container
 
-| Name         | Type   | Required | Description                                                                       |
-| ----------   | ------ | -------- | --------------------------------------------------------------------------------- |
-| classes      | string | No       | Classes to add to the timeline's container.                                       |
-| attributes   | object | No       | HTML attributes (for example data attributes) to add to the timeline's container. |
-| headingLevel | object | No       | Defaults to 2 (`<h2>`).                                                           |
+| Name         | Type    | Required | Description                                                                       |
+| ----------   | ------  | -------- | --------------------------------------------------------------------------------- |
+| classes      | string  | No       | Classes to add to the timeline's container.                                       |
+| attributes   | object  | No       | HTML attributes (for example data attributes) to add to the timeline's container. |
+| headingLevel | numeric | No       | Defaults to 2 (`<h2>`).                                                           |
```

Comment:
Sorry, I obviously didn't read this properly last night.  I thought we were usding the same description for headingLevel as from side navigation? which is: 

> A number for the heading level. Defaults to 4 (`<h4>`)

If you decided to change it that's fine, just wanted to check.